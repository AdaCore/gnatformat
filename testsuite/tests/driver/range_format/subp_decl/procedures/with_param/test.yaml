description: SubpDecl formatting for line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 25",
    "-EL 26",
    "--pipe",
  ]
