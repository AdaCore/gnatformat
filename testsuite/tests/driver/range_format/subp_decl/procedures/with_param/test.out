test.ads(<SubpDecl ["Right_Indent"] test.ads:25:4-25:51>) - 25:1-25:51
^
   procedure Right_Indent (Margin : out Line_Size);$

